### ISSUE TITLE
Interaction with attrs

### ISSUE THREAD

**Issue by robertschweizer** [fcfangcc/pylint-pydantic#29]
## Problem description

Running below example with `--load-plugins=pylint_pydantic` leads to this error:

```
test_repro.py:10:4: C0202: Class method _validate should have 'cls' as first argument (bad-classmethod-argument)
````

Without the `pylint_pydantic` plugin enabled, this error is not reported.

attrs documentation uses examples where validators accept `self`. Whether they should usually be classmethods and accept `cls` is a different discussion. This plugin should not interact with unrelated code that has nothing to do with pydantic.

## Steps to reproduce the problem

```py
from attrs import define, field

@define
class Coordinates:
    x: int = field()
    y: int = field()

    @x.validator
    @y.validator
    def _validate(self, _, value):
        assert value >= 0

Coordinates(x=1, y=2)
```

## Specifications like the version of the project, operating system, or hardware
* pylint version: 3.0.2
* pydatic version: 2.5.1
* python version: 3.10
* pylint-pydantic version: 0.3.0
* other more: ...

**Comment by fcfangcc** [comment]
Identifying  which module **validator** comes from is difficult. 😢  

I'll give it a try

**Comment by robertschweizer** [comment]
I can confirm that the fix is working, thanks a lot @fcfangcc!
